packages/odyssey-react-mui/src/BackgroundContext.tsx

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { createContext, useContext, ReactNode } from "react";
+
+export type BackgroundType = "white" | "gray";
+
+const BackgroundContext = createContext<BackgroundType>("white");
```

Comment:
We can't use words like "white" and "gray". If we have Dark mode, they won't make sense.

That's why I was suggesting "high contrast" and "low contrast". Maybe we can figure out better words.

Reply:
Great callout. I've updated it to `LowContrast` (gray) / and `highContrast` (white) which I think will be more scalable if/when we add dark mode since it will still be relevant for styles.